doc/harmony-ap_installation_guide.md

```diff
@@ -227,4 +230,35 @@ During the installation process, multiple random passwords are generated.
 | Content encryption keystore (`/etc/harmony-ap/ap-keystore.jks`) password | Configuration file: `/etc/harmony-ap/domibus.properties`<br /><br />Properties: `domibus.security.keystore.password` and `domibus.security.key.private.password`. Content of this keystore can be changed using the administrative UI. |
 | Content encryption truststore (`/etc/harmony-ap/ap-truststore.jks`) password | Configuration file: `/etc/harmony-ap/domibus.properties`<br /><br />Properties: `domibus.security.truststore.password`. Content of this keystore can be changed using the administrative UI. |
 | TLS keystore (`/etc/harmony-ap/tls-keystore.jks`) password | Configuration file: `/etc/harmony-ap/conf/server.xml`<br /><br />Property: `keystorePass` |
-| TLS truststore (`/etc/harmony-ap/tls-truststore.jks`) password | Configuration file: `/etc/harmony-ap/conf/server.xml`<br /><br />Property: `truststorePass` |
\ No newline at end of file
+| TLS truststore (`/etc/harmony-ap/tls-truststore.jks`) password | Configuration file: `/etc/harmony-ap/conf/server.xml`<br /><br />Property: `truststorePass` |
+
+### 2.11 Log Files
+
+The Access Point application log files are located in the `/var/log/harmony-ap/` directory.
+
+## 3 Version Upgrade
```

Comment:
I wonder if there is a way we could have it so the installation restarts and reloads the service configuration. At the moment, one of the dangers is that when we release a new package to the main repository and the user doesn't notice it, runs an `apt upgrade`, the AP will be unreachable until they manually restart the service.

Reply:
That's a good point. I updated the packaging scripts to stop the `harmony-ap` service before the upgrade and to start it again after the upgrade if the service has been enabled. However, if the service is disabled then the user must restart the service manually.